examples/oneview_server_hardware_type.yml

```diff
@@ -66,23 +65,26 @@
         config: "{{ config }}"
         state: present
         data:
-          name: '{{ server_hardware_type_name_renamed }}'
-          newName: "{{ server_hardware_types[0]['name'] }}"
+            name: '{{ server_hardware_type_name_renamed }}'
+            newName: "{{ server_hardware_types[0]['name'] }}"
       delegate_to: localhost
 
+    # This is only supported on appliances which support rack mounted servers.
+    - name: Remove the Server Hardware to be able to delete the Server Hardware Type (Prerequisite)
+      oneview_server_hardware:
+        config: "{{ config }}"
+        state: absent
+        data:
+            name : '{{ server_hardware_hostname }}'
+      delegate_to: localhost
+      when: variant == 'C7000'
 
-#    - name: Remove the Server Hardware to be able to delete the Server Hardware Type (Prerequisite)
-#      oneview_server_hardware:
-#        config: "{{ config }}"
-#        state: absent
-#        data:
-#            name : '{{ server_hardware_hostname }}'
-#      delegate_to: localhost
-#
-#    - name: Delete the Server Hardware Type
-#      oneview_server_hardware_type:
-#        config: "{{ config }}"
-#        state: absent
-#        data:
-#            name: '{{ server_hardware_type_name_renamed }}'
-#      delegate_to: localhost
+    # This is only supported on appliances which support rack mounted servers.
+    - name: Delete the Server Hardware Type
+      oneview_server_hardware_type:
+        config: "{{ config }}"
+        state: absent
+        data:
+            name: '{{ server_hardware_type_name_renamed }}'
```

Comment:
Actually in the previous task we changed the server hardware name to its original name......so it should be server_hardware_type_name....right?